public/app/ui/components/collections/memorials/list-result.html

```diff
@@ -1,4 +1,9 @@
 <div class="list-result list-result-memorials">
     <span class="pull-left" data-bind="component: displayFor('MediaThumbnail', result)"></span>
-    <a href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayForLabelField(result)"></a>
+
+    <a class="result-title" href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayForLabelField(result)"></a>
+
+	<div class="inline-results"  data-bind="component: displayFor('idno', result)"></div>
```

Comment:
I don't think the accession number should be here.  It either shouldn't be in any of the results, or it should be in a consistent position (across all the collections), possibly to the bottom right, but muted text.